server-side/controllers/user.controller.js

```diff
@@ -15,7 +15,8 @@ export const getUsers = async (req, res,next) => {
 };
 
 export const getUserById = async (req, res,next) => {
-  const id = req.params.id;
+  let id = req.params?.id;
+  id=id.split('').slice(1).join('').trim();
```

Comment:
why do you need this formatting?

Reply:
same as above.
id is extracted like this form example
":123456789"

Reply:
so i remove it.